client/components/signup-form/index.jsx

```diff
@@ -591,6 +595,20 @@ class SignupForm extends Component {
 			<div className={ classNames( 'signup-form', this.props.className ) }>
 				{ this.getNotice() }
 
+				{ this.props.isSocialSignupEnabled &&
+					this.props.isSocialSignupFirst && (
+						<Card className="signup-form__social logged-out-form">
+							<SocialSignupForm
+								handleResponse={ this.props.handleSocialResponse }
+								socialService={ this.props.socialService }
+								socialServiceResponse={ this.props.socialServiceResponse }
+							/>
+							<p>
+								{ preventWidows( this.props.translate( 'Or sign up with your email address.' ) ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Or sign up with your email address:' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).